### PR TITLE
drop grid image

### LINKED ISSUE #418: Drop `grid-dark.png`
Drop `grid-dark.png` and use CSS to produce the same effect. It is used in the footer images.

### PR DESCRIPTION
**Pull request by Bhavyajain21**
/claim #418 
/fixes #418

### CODE REVIEW

**Comment by Bhavyajain21** [comment]
image

File: src/components/shared/Footer.tsx [hunk 1]
```diff
@@ -7,12 +7,7 @@ import {pageLinks} from "@site/src/constants/routes"
 const Footer = (): JSX.Element => {
   const year = new Date().getFullYear()
   return (
-    <footer className="bg-tailCall-dark-700 pt-SPACE_10 pb-SPACE_05 flex flex-col items-center justify-center gap-SPACE_10 w-full relative px-SPACE_04">
-      <img
-        src={require("@site/static/images/about/grid-dark.png").default}
-        alt="grid background"
-        className="absolute inset-0 w-full h-full"
-      />
+    <footer className="bg-[#1C1D1F] grid-background pt-SPACE_10 pb-SPACE_05 flex flex-col items-center justify-center gap-SPACE_10 w-full relative px-SPACE_04">
```
Comment (tusharmath) [inline src/components/shared/Footer.tsx]: This grid a bit different :)

Comment (tusharmath) [reply src/components/shared/Footer.tsx]: Check the design https://www.figma.com/design/mMLCy6XZinopm6QrEqOovP/Tailcall-Website-Redesign---Shared?node-id=2913-221753&t=ImOZMgbhPu8a9kMe-0

Comment (Bhavyajain21) [reply src/components/shared/Footer.tsx]: Updated!

**Comment by Bhavyajain21** [comment]
image

**Comment by amitksingh1490** [comment]
Pr not matching the original design moving this to draft

**Comment by Bhavyajain21** [comment]
On it! @amitksingh1490

**Comment by Bhavyajain21** [comment]
The PR is ready for review! @amitksingh1490

**Comment by amitksingh1490** [comment]
closing a favour of https://github.com/tailcallhq/tailcallhq.github.io/pull/421

**Pull request closed without merging**